src/main/scala/no/ndla/articleapi/model/domain/Article.scala

```diff
@@ -16,8 +16,8 @@ import org.json4s.FieldSerializer._
 import org.json4s.native.Serialization._
 import scalikejdbc._
 
-case class Article(id: Option[Long],
-                   revision: Option[Int],
+case class Article(_id: Option[Long],
+                   _revision: Option[Int],
```

Comment:
For å være helt ærlig, så synes påvirkingen av denne endringen forkludrer koden mer enn `article.id.get` gjorde i utgangspunktet. Bruken av `_id` og `_revision` rundt omkring i konverterere, copy og tester synes jeg er mindre pen. Usikker på om dette er verdt det.